data/transactions/logic/eval.go

```diff
@@ -248,6 +248,23 @@ type boxRef struct {
 	name string
 }
 
+// EvalConstants contains data that opcodes rely on during runtime.
```

Comment:
nit: update doc to match new name/purpose?

Reply:
you mean renaming to `EvalChanges` to be symmetric to the resp body?

Reply:
No I mean that "contains data that opcodes rely on during runtime" was a good description for the old `EvalOpParams`, but not for `EvalConstants`. I expect to see something like "contains constants used during evaluation"

Reply:
see [ResultEvalConstants -> ResultEvalOverrides](https://github.com/algorand/go-algorand/pull/5247/commits/02d7a219ba0c86e2f8fdb734a9443ec9c2ce4206), I update the docstring a bit.